alfresco-solr-query-analytics/publish.gradle

```diff
@@ -10,7 +10,7 @@ publishing {
         all {
             pom {
                     url = 'https:/github.com/xenit-eu/alfresco-solr-query-analytics'
-                    name = "alfresco-solr-query-analytics-6x"
+                    name = "alfresco-solr-query-analytics"
```

Comment:
Wasn't sure about this; maybe we want to keep the version number in the name?

Reply:
In which case I would go more specific than '7x'

Reply:
Let's discuss. We can be consistent with the solr-patch amp

Reply:
to be changed to 7x